ir_cdk_stacks/in_lam_01/response_lambda.py

```diff
@@ -54,6 +54,17 @@ def lambda_handler(event, context):
             UserName=uName
         )
         print(response)
+
+        # This code should work, but times out due to function not being properly created yet
+        # Can't properly debug as don't have access to CloudWatch Logs!!!
+        # Can't add sleep as otherwise costs ramp up
+        # Once off CloudWatch Event?
+        if eventName.startswith("CreateFunction"):
```

Comment:
If it does not produce reliable results, best not to automate it. Instead, you should advise what to do in the run book.